camel-idea-plugin/src/main/java/org/apache/camel/idea/annotator/CamelSimpleAnnotator.java

```diff
@@ -49,7 +50,7 @@ boolean isEnabled() {
      */
     void validateText(@NotNull PsiElement element, @NotNull AnnotationHolder holder, @NotNull String text) {
         boolean hasSimple = text.contains("${") || text.contains("$simple{");
-        if (hasSimple && CamelIdeaUtils.isCamelSimpleExpression(element)) {
+        if (hasSimple || CamelIdeaUtils.isCamelSimpleExpression(element)) {
```

Comment:
Yes but its still wrong. We need to add special support for log. Its only for simple / <simple> currently.

Otherwise you end up validating all kind of non Camel stuff as well.

My PR will correct this and have a TODO for log